bioptim/limits/penalty.py

```diff
@@ -457,27 +457,30 @@ def custom(penalty, ocp, nlp, t, x, u, p, **parameters):
             parameters["penalty"] -> Index of the penalty (integer), parameters.weight -> Weight of the penalty
             (float)
             """
-            val = penalty.custom_function(ocp, nlp, t, x, u, p, **parameters)
-            if isinstance(val, tuple):
-                if "min_bound" in parameters or "max_bound" in parameters:
-                    raise RuntimeError(
-                        "You cannot have non linear bounds for custom constraints and min_bound or max_bound defined"
-                    )
-                min_bound = val[0]
-                max_bound = val[2]
-                val = val[1]
-
+            nl_bound = True
```

Comment:
That is merely a technicality, but I feel that nl_bound is less common than defining a linear bound. So I feel that `has_linear_bound` to be a better name.
Also, if none of them are defined, the name `has_nl_bound` suggest that there is an nl_bound while there is actually no bounds. 

Apart from that, this seems good